app/views/welcome/_unread_msg.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id= "<%= chat.id%>" class= "unread-msg-div" data-unread-count= <%= results[chat.id]%>>
```

Comment:
same as above. istead of just a number use something like id="unread-message-<%= chat.id%>" . Because you could have problems if there are 2 elements with the same ID